.idea/codeStyles/Project.xml

```diff
@@ -1,22 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
-      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
I don't see why this is pushed to main, is it really needed :D Maybe we can add it to git ignore if you automatically add everything to commit.

Reply:
I don't know neither... Each time I build locally, this is deleted...